### PR TITLE
Korean translation for React section

### PR DESCRIPTION
**Pull request by katepark1009**
Hi!😄 
I translated React section for `Intro to Storybook` into Korean. 
I Had lot of fun while I was working on it.
Please proofread and merge this PR.

### CODE REVIEW

**Comment by edpark722** [comment]
Great translation work!

**Comment by jonniebigodes** [comment]
@katepark1009 thank you very much for this for the time and effort you put into this. And glad that you had a lot of fun working on it 👍 

Wow this is really something else. 

I'm going to ping @domyen so that he can reach out to the community and see if we can get this proofread and merged. 

Stay safe.

**Comment by katepark1009** [comment]
@jonniebigodes, @domyen 
If you have a trouble to find someone to proofread the translation, I can reach out to Korean programmers. 
Please let me know. 
So excited that my work could help Korean users in the future. 
Have a great day!

**Comment by domyen** [comment]
@katepark1009 that'd be lovely if you reached out to them. 🙇  

I tried in the SB community but it doesn't seem like there are many and I don't know many Korean devs myself. 

Again, thanks so much for your contribution!

**Comment by katepark1009** [comment]
@domyen 
Cool! What would be the process for proofreading? 😃 
Should I mention them here? or can I just let them read my translations and get revised?

**Comment by domyen** [comment]
Link them to your PR then they can make comments and suggestions via
GitHub. Once y'all think it's good to go, mention @jonniebigodes or me and
we'll merge it. The process is light :)

>

**Comment by katepark1009** [comment]
@jinho6225 will proofread. Big thanks to him!

File: content/intro-to-storybook/react/kr/screen.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+우리는 가장 아래에 해당하는 `Task`로부터 시작하여, `TaskList`로 진행하였고 이제 전체 화면을 구성하는 UI를 완성하였습니다. `InboxScreen`은 중첩된 컨테이너 컴포넌트를 수용하고 그에 수반하는 스토리들을 포함하고 있습니다.
+
+<video autoPlay muted playsInline loop style="width:480px; height:auto; margin: 0 auto;">
+  <source
+    src="/intro-to-storybook/component-driven-development-optimized.mp4"
+    type="video/mp4"
+  />
+</video>
+
+[**컴포넌트 기반 개발**](https://blog.hichroma.com/component-driven-development-ce1109d56c8e)은 컴포넌트의 상위 계층으로 올라감에 따른 복잡성을 점진적으로 확장할 수 있도록 해줍니다. 이것의 이점 중 하나는 보다 개발 과정에 집중할 수 있으며 가능한 모든 UI 순열의 적용 범위가 늘어난다는 것입니다. 간단히 말하면, 컴포넌트 기반 개발(CDD)은 더 높은 품질과 복잡성을 가진 사용자 인터페이스를 만들 수 있도록 도와줍니다.
```
Comment (jonniebigodes) [inline content/intro-to-storybook/react/kr/screen.md:178]: @katepark1009 when you can, please adjust the link from https://blog.hichroma.com/component-driven-development-ce1109d56c8e to https://www.componentdriven.org/

File: content/intro-to-storybook/react/kr/simple-component.md [hunk 2]
```diff
@@ -0,0 +1,259 @@
+---
+title: '간단한 컴포넌트 만들기'
+tocTitle: '간단한 컴포넌트'
+description: '간단한 컴포넌트를 독립적으로 만들어봅시다'
+commit: '3d9cd8c'
+---
+
+우리는 [컴포넌트 기반 개발(Component-Driven Development)](https://blog.hichroma.com/component-driven-development-ce1109d56c8e) (CDD) 방법론에 따라 UI를 만들어 볼 것입니다. 이는 컴포넌트로부터 시작하여 마지막 화면에 이르기까지 상향적으로 UI를 개발하는 과정입니다. CDD는 UI를 구축할 때 직면하는 복잡성을 확장하는데 도움을 줍니다.
+
```
Comment (jonniebigodes) [inline content/intro-to-storybook/react/kr/simple-component.md:9]: @katepark1009 when you're able please adjust the link "Component-Driven Development" to https://www.componentdriven.org/

File: content/intro-to-storybook/react/kr/using-addons.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+이제 하단 창의 "Action Logger" 탭 옆에 새로운 "Knobs" 탭이 나타납니다.
+
+[여기](https://github.com/storybooks/storybook/tree/master/addons/knobs#object)에 설명된 것처럼, `object` knob는 라벨과 "default" 객체를 매개변수로 받아들입니다.
+라벨은 애드온 패널의 왼쪽에 표시됩니다. 전달한 객체는 편집 가능한 JSON blob으로 표시됩니다. 유효한 JSON 객체를 보내는 한, 컴포넌트가 객체에 전달된 데이터를 바탕으로 조정될 것입니다!
+
+## Storybook의 범위를 넓혀주는 애드온
+
+이제 Storybook이 뛰어난 [CDD 개발환경](https://blog.hichroma.com/component-driven-development-ce1109d56c8e)을 제공해줄 뿐만 아니라, 상호작용을 할 수 있는 문서 자료 또한 제공합니다. PropTypes는 훌륭하지만, 디자이너 또는 컴포넌트의 코드에 익숙하지 않은 사람도 knobs 애드온이 구현된 Storybook을 통해서라면 컴포넌트가 어떻게 작동하는지를 쉽게 이해하실 수 있습니다.
+
```
Comment (jonniebigodes) [inline content/intro-to-storybook/react/kr/using-addons.md:111]: @katepark1009 when you're able please adjust the link to "CDD" link to https://www.componentdriven.org/

File: content/intro-to-storybook/react/kr/test.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+
+이는 새로운 배경색을 연출할 것입니다.
+
+![task의 배경이 변화함](/intro-to-storybook/chromatic-task-change.png)
+
+아래 파일을 추가해주세요:
+
+```bash
+git add src\components\Task.js
+```
```
Comment (jonniebigodes) [inline content/intro-to-storybook/react/kr/test.md:68]: @katepark1009 when you're able change the snippet from "git add src\components\Task.js" to "git add .". With a recent change i made this was added and could lead to confusion to users that use mac or linux

**Review by jonniebigodes — CHANGES_REQUESTED** [review]
@katepark1009 left some comments for some items that need to be fixed when you can. 

Stay safe

**Comment by katepark1009** [comment]
@jonniebigodes Great. Thanks for the updates. I will update it ASAP.
and also @heegupark will proofread the translations. Thank you so much Heegu👍

File: content/intro-to-storybook/react/kr/screen.md (hunk 1 above)
Comment (katepark1009) [reply content/intro-to-storybook/react/kr/screen.md]: @jonniebigodes All CDD links got updated. Thank you!

File: content/intro-to-storybook/react/kr/test.md (hunk 4 above)
Comment (katepark1009) [reply content/intro-to-storybook/react/kr/test.md]: Updated. Thanks!

File: content/intro-to-storybook/react/kr/conclusion.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+- [**시각적 테스팅 핸드북**](https://www.chromatic.com/book/visual-testing-handbook)에서는
+  Storybook을 컴포넌트의 시각적 테스트에 사용하는 것에 대해 알아봅니다. 31페이지 분량의 무료 전자책.
+
+- [**Storybook Discord 채널**](https://discord.gg/UUt2PJb)로 Storybook 커뮤니티와 연결될 수 있습니다. Storybook 사용자를 위한 도움을 주고받으세요.
+
+- [**Storybook 블로그**](https://medium.com/storybookjs)는 UI 개발을 위한 workflow를 간소화할 수 있는 최신 릴리즈와 기능들을 보여줍니다.
+
+## LearnStorybook.com은 누가 만들었나요?
+
+내용, 코드, 제작은 [Chromatic](http://blog.hichroma.com/)이 기여해주셨습니다. 튜토리얼은 Chroma의 인기 있는 [GraphQL + React 튜토리얼 시리즈](https://blog.hichroma.com/graphql-react-tutorial-part-1-6-d0691af25858)에서 영감을 얻었습니다.
```
Comment (jinho6225) [inline content/intro-to-storybook/react/kr/conclusion.md:31]: 내용, 코드, 제작은 [Chromatic](http://blog.hichroma.com/)이 기여해주셨습니다.
=>
내용, 코드, 제작은 [Chromatic](http://blog.hichroma.com/)에서 기여해주셨습니다.

File: content/intro-to-storybook/react/kr/contribute.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
+---
+title: '기여하기'
+description: 'Storybook을 세상과 함께 나눌 수 있도록 도와주세요'
+---
+
+'Storybook 배우기'에 기여해주세요! 문법이나 구두점 같은 작은 것이라면, pull request를 해주세요. 보다 큰 변화가 필요하다면 토론을 위해 [이슈를 추가](https://github.com/chromaui/learnstorybook.com/issues)해주세요.
+
+'Storybook 배우기'는 주로 커뮤니티에 의해 만들어지고 유지되어 왔기 때문에, 이를 최신 상태로 유지하고 시간이 지남에 따라 다듬기 위해서는 많은 분의 도움이 필요합니다. 모든 도움에 감사드릴 것입니다!
```
Comment (jinho6225) [inline content/intro-to-storybook/react/kr/contribute.md:8]: 'Storybook 배우기'는 주로 커뮤니티에 의해 만들어지고 유지되어 왔기 때문에, 이를 최신 상태로 유지하고 시간이 지남에 따라 다듬기 위해서는 많은 분의 도움이 필요합니다. 모든 도움에 감사드릴 것입니다!
=>
'Storybook 배우기'는 주로 커뮤니티에 의해 만들어지고 유지되어 왔기 때문에, 이를 최신 상태로 유지하고 거친 가장자리가 시간이 지남에 따라 부드럽게 처리되도록 모든 사람의 도움이 필요합니다. 모든 도움에 감사드립니다!

**Comment by jonniebigodes** [comment]
@katepark1009 one small item when you take care of the updates for 6.0, now that the pull request i've mentioned is merged, I would like for you to revert the comments in the code snippets and leave them in english if you don't mind. I know that translating said snippets brings the translation "closer to home" (pardon the bad pun), but general rule of thumb it's best to avoid translate them.

Can't wait to see this updated to 6.0 and merged. 

Stay safe

**Comment by katepark1009** [comment]
@jonniebigodes 
Hi! 
I noticed it got merged. I've been working on revising & updating it with 6.0 version at the same time.
While I was translating the 6.0, I actually suggested to our company to migrate to Storybook 6. 
New version with docs, controls looks awesome and fun to work with.
and I can revert the comments in code snippets, of course!

Stay safe and thank you for letting me know. 👍

**Comment by jonniebigodes** [comment]
Hi @katepark1009 , that's great to hear. I'll be on the lookout and see when it's updated, so that it can be proofread and we get it merged. And wow, that's awesome, i hope your company adopts Storybook 😁 ! And indeed, Docs and Controls are fantastic addons to work with.

Stay safe

**Comment by katepark1009** [comment]
Hi @jonniebigodes.
I finished the translation with the updates from the reviewers.
Also It's up-to-date, including the latest Storybook 6.0 changes.
Could you review it? 
Thank you and glad that I can contribute to the community.😸 

Stay safe!

**Comment by jonniebigodes** [comment]
@katepark1009 took a look at it and it's good to from your end. I'm going to merge it. Can't thank you enough for the time and effort you put into this. And it goes without saying thank you as well to @jinho6225 and @heegupark for helping out in proofreading it.

Stay safe all of you.

**Pull request merged**